### PR TITLE
Fix/Improve the console logging when the server starts

### PR DESCRIPTION
**Pull request by sneridagh**
image

How about this?

### CODE REVIEW

File: src/start-server.js [hunk 1]
```diff
@@ -27,7 +27,10 @@ export default () => {
         console.log(
           `Using internal proxy: ${app.publicURL} -> ${app.devProxyToApiPath}`,
         );
-      console.log(`🎭 Volto started at ${app.publicURL} 🚀`);
+      if (!process.env.RAZZLE_PUBLIC_URL)
+        console.log(`Current guessed public URL: ${app.publicURL}`);
```
Comment (tiberiuichim) [inline src/start-server.js:31]: I'm not a fan of "guessed", but I can't really come up with a better suggestion. I think the fact that "things are tricky/complicated here" is a smell.

Comment (sneridagh) [reply src/start-server.js:31]: Just `Current public URL`? No info at all? We could put additional info under a `debug` lib flag.

Comment (sneridagh) [reply src/start-server.js]: done in the last commits

**Comment by sneridagh** [comment]
image

**Review by tiberiuichim — APPROVED** [review]

**Pull request merged**